lib/platform/bitbucket/index.ts

```diff
@@ -467,15 +468,17 @@ async function closeIssue(issueNumber: number): Promise<void> {
 
 export function massageMarkdown(input: string): string {
   // Remove any HTML we use
-  return smartTruncate(input, 50000)
-    .replace(
-      'you tick the rebase/retry checkbox',
-      'rename PR to start with "rebase!"'
-    )
-    .replace(/<\/?summary>/g, '**')
-    .replace(/<\/?details>/g, '')
-    .replace(new RegExp(`\n---\n\n.*?<!-- rebase-check -->.*?\n`), '')
-    .replace(/\]\(\.\.\/pull\//g, '](../../pull-requests/');
+  return sanitizeMarkdown(
```

Comment:
I think that sanitizing needs to happen first, otherwise it could increase the size beyond the truncation limit.